src/plugins/dashboard/public/plugin.tsx

```diff
@@ -343,7 +343,9 @@ export class DashboardPlugin
       this.dashboardItemCreators.push(creator);
     };
 
-    const dashboardCreate = (history) => history.push(DashboardConstants.CREATE_NEW_DASHBOARD_URL);
+    const dashboardCreate = () => {
+      window.location = core.http.basePath.prepend(DashboardConstants.CREATE_NEW_DASHBOARD_URL);
```

Comment:
Works as bench-tested.   Should also be defacto tested w/ current OSD tests.

Reply:
but `core.http.basePath` is probably `/abc`, the constant is `/create`
https://github.com/opensearch-project/OpenSearch-Dashboards/blob/7e7a5129b47a83f96e21df887be405acba5cfd5b/src/plugins/dashboard/public/dashboard_constants.ts#L33

there is no `/app/dashboards#` in between, i'm not sure why it can work

does it work for you locally?

Reply:
Yes, `.prepend` does the concatenation with basepath.   Works locally.

Reply:
@pjfitzgibbons, @joshuali925 is right. `.prepend` does not work as expected when `/app/dashboards#` is the path. I was able to reproduce the error locally.

Steps to repo:
1. Clean install with no plugins
2. Go to : http://localhost:5601/app/dashboards#/list?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:now-15m,to:now))
3. Click on the create dashboards button
4. the route is: `http://localhost:5601/create` and is a 404

Reply:
@ashwin-pc Thank you for your help today.  PR is updated and now should correctly navigate to Dashboard Create